Sources/TSCTestSupport/Product.swift

```diff
@@ -100,6 +100,9 @@ extension Product {
         // Unset the internal env variable that allows skipping certain tests.
         environment["_SWIFTPM_SKIP_TESTS_LIST"] = nil
 
+        // Skip building PackageDescription libraries at runtime.
+        environment["SKIP_BUILDING_PD_AT_RUNTIME"] = "1"
```

Comment:
In what case would one want to skip building at runtime?

Reply:
Since SwiftPM tests usually run in parallel, it's not safe to build the libraries at runtime :/ This basically means the libraries must be built before the tests are run but I don't see a way around that right now. 

@harlanhaskins promised (😤) to add a flag in Swift compiler to force load a module by path so we might be able to just build the package description libraries as part of the actual build process.